chrome/content/zotero/preferences/quickCopySiteEditor.xul

```diff
@@ -46,6 +47,7 @@
 				var io = window.arguments[0];
 				io.domain = document.getElementById('zotero-quickCopy-domain').value;
 				io.format = document.getElementById('zotero-quickCopy-menu').value;
+				io.locale = document.getElementById('locale-menu').label !== '' ? document.getElementById('locale-menu').value : '';
```

Comment:
So, what I currently do is: when the user selects an export translator, a new menuitem is added at the top of the list. This menuitem's label is an empty string, but its value is set to the value of the previously selected menuitem. The menulist is then disabled. When the user selects a style and the menulist has been disabled, I read out the value of the blank-label menuitem, remove it, and restore the original selection. See https://github.com/zotero/zotero/pull/623/files#diff-9e2de9af59f63dea6fb7286bcf2bfa1bR155

It's why I test whether the label is blank here. (alternatively, I could test if the menulist is disabled)